### DISCUSSION TITLE
Show basic info on profile but not activity

### DISCUSSION THREAD

**Discussion by lancejpollard** [community/community#73518] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I would like to have my profile show:

- Left sidebar stuff (email and basic links/info, badges, and orgs I belong to)
- Readme
- Pinned repos
- Repos

But I would like to hide the activity chart and timeline of activity.

Is that possible? I seem to only have 2 options:

1. Hide everything except for my readme. Hides all sidebar info as well.
2. Show everything.

Maybe I should just use the readme to recreate the pinned repos then? And just forget about showing the badges and dynamically the orgs?

**Comment by reussio** [comment, marked as answer]
Yes, it's possible to selectively hide or show certain elements on your GitHub profile, but customization options are limited. GitHub doesn't allow users to specifically hide the activity graph or the activity timeline via the user interface. The options you've found are typically what GitHub offers:

1. **Hide everything except the readme:** This is done by setting the visibility of your activities to "private," which results in all sidebar information being hidden except for the readme.

2. **Show everything:** This option will make your entire profile public, including the activity graph and timeline.

If you still want to customize your profile despite these limitations, you could:

- **Use the readme:** You can indeed use your profile readme to manually display information like Pinned Repositories. You could embed images of your repositories, links to them, and descriptions that would function like your "Pinned Repos." It requires a bit more upkeep but provides you with a lot of flexibility in how you present your work.

- **Badges and Organizations:** You could include badges and a list of organizations you're a part of in your readme by using appropriate images and links.

To do this, you would insert images and links using the correct markdown syntax in your readme. Here's an example of markdown you could use in your readme:

```
### Badges
![Badge](URL_to_badge)

### Organizations
- [Organization 1](URL_to_organization_1)
- [Organization 2](URL_to_organization_2)
- [Organization 3](URL_to_organization_3)

### Pinned Repositories
[![Repo Name](URL_to_repo_image)](URL_to_repo)
```
